templates/service-template.yml

```diff
@@ -165,7 +165,12 @@ parameters:
 - name: DYNAMIC_SCALING_CONFIG
   displayName: Dynamic Scaling configuration
   description: "YAML content containing a map of the dynamic scaling configuration for each instance type"
-  value: "{developer: {reserved_streaming_units: 1, compute_nodes_config: {max_compute_nodes: 3}}, standard: {reserved_streaming_units: 1, compute_nodes_config: {max_compute_nodes: 9}}}"
+  value: "{developer: {compute_nodes_config: {max_compute_nodes: 3}}, standard: {compute_nodes_config: {max_compute_nodes: 9}}}"
+
+- name: NODE_PREWARMING_CONFIG
+  displayName: Node prewarming configuration
+  description: "YAML content containing a map of the node prewarming configuration for each instance type"
+  value: "{}"
```

Comment:
This equates to pre-warming being disabled completely for each instance type.

It can also be disabled on per instance type level by setting reserved_streaming_units to `0`

Reply:
cc @pb82 as mentioned in the internal call